packages/kit/src/core/utils.js

```diff
@@ -70,7 +70,14 @@ export function list_files(dir, filter) {
 	function walk(current) {
 		for (const file of fs.readdirSync(path.resolve(dir, current))) {
 			const child = path.posix.join(current, file);
-			if (fs.statSync(path.resolve(dir, child)).isDirectory()) {
+			let stat;
+			try {
+				stat = fs.statSync(path.resolve(dir, child));
+			} catch {
+				continue;
```

Comment:
I think it'd be nice to print a warning letting the user know they have a broken symlink so that they can remove/fix it

Reply:
Logging it as error/warning sounds good. On it

Reply:
Something like this?

It will also print a stacktrace, which is very handy for figuring out what is going on. But I don't know what the policy of SvelteKit is regarding user facing messages.